commands/remove/index.js

```diff
@@ -0,0 +1,149 @@
+
+    this.dirs = new Set(this.options.globs.map(fp => path.resolve(this.project.rootPath, fp)));
+
+    let chain = Promise.resolve();
+
+    chain = chain.then(() => getFilteredPackages(this.packageGraph, this.execOpts, this.options));
+    chain = chain.then(filteredPackages => {
+      this.filteredPackages = filteredPackages;
+    });
+
+    chain = chain.then(() => this.collectPackagesToChange());
+    chain = chain.then(packagesToChange => {
+      this.packagesToChange = packagesToChange;
+    });
+
+    return chain.then(() => {
+      const proceed = this.packagesToChange.length > 0;
+
+      if (!proceed) {
+        this.logger.warn(`No packages found where ${this.dependencyToRemove} can be removed.`);
+      }
+
+      return proceed;
+    });
+  }
+
+  execute() {
+    const numberOfPackages = `${this.packagesToChange.length} package${
+      this.packagesToChange.length > 1 ? "s" : ""
+    }`;
+
+    this.logger.info("", `Remove ${this.dependencyToRemove} from ${numberOfPackages}`);
+
+    let chain = Promise.resolve();
+
+    chain = chain.then(() => this.removePackage());
+
+    // if (this.options.bootstrap !== false) {
+    //   chain = chain.then(() => {
+    //     const argv = Object.assign({}, this.options, {
+    //       args: [],
+    //       cwd: this.project.rootPath,
+    //       // silence initial cli version logging, etc
+    //       composed: "add",
+    //     });
+
+    //     return bootstrap(argv);
+    //   });
+    // }
+
+    return chain;
+  }
+
+  collectPackagesToChange() {
+    let result = this.filteredPackages;
+
+    // Skip packages that are not selected by dir globs
+    if (this.dirs.size) {
+      result = result.filter(pkg => this.dirs.has(pkg.location));
+    }
+
+    // Skip packages that has no such dependency
+    result = result.filter(pkg => this.hasDependency(pkg));
+
+    return result;
+  }
+
+  hasDependency(pkg) {
+    return ["dependencies", "devDependencies", "peerDependencies"].some(dependencyType => {
+      const deps = pkg.get(dependencyType);
+      return deps && this.dependencyToRemove in deps;
+    });
+  }
+
+  removePackage() {
+    let chain = Promise.resolve();
+    chain = chain.then(() => this.removeFromManifest());
+
+    chain = chain.then(() => {
+      const isDependencyToRemoveExternal = !this.packageGraph.has(this.dependencyToRemove);
+      if (isDependencyToRemoveExternal) {
+        this.removeExternal();
+      } else {
+        this.removeSymlink();
+      }
+    });
+
+    return chain;
+  }
+
+  removeFromManifest() {
+    return pMap(this.packagesToChange, pkg => {
+      if (pkg.dependencies) {
+        delete pkg.dependencies[this.dependencyToRemove];
+      }
+      if (pkg.devDependencies) {
+        delete pkg.devDependencies[this.dependencyToRemove];
+      }
+      if (pkg.peerDependencies) {
+        delete pkg.peerDependencies[this.dependencyToRemove];
+      }
+      return pkg.serialize();
+    });
+  }
+
+  removeSymlink() {
```

Comment:
We're not removing the symlink, but the directory. In any case, I don't think we need this. Just bootstrap after serializing the modified dependencies.